### DISCUSSION TITLE
New Dashboard

### DISCUSSION THREAD

**Discussion by aamiel1** [eusonlito/GPS-Tracker#54] [Ideas]
The dashboard isn't merely an ultimate control panel; rather, it should present data summaries in visually designed cubes. These summaries may include: the number of registered vehicles, the count of devices, the volume of repairs in liters since the beginning of the month, the number of treatments, and the number of trips. Additionally, it should display the total kilometers traveled per day, per week, and for the last month. Furthermore, it should feature a table of recent alerts and incorporate a map showing the current location of vehicles (last known point only).

Moreover, it should offer an option for automatic screen refresh at defined intervals of 5, 10, or 20 minutes.

Thanks,

**Comment by eusonlito** [comment]
My idea is that the dashboard should be as simple as possible, as it is now I think it is correct.

But maybe a new statistics panel could be added for all those cases.

In any case, in principle this platform is more intended for private use than for business use with fleets of vehicles, hence the simplicity.

My idea is to keep this line as a simple alternative to Traccar.

**Comment by aamiel1** [comment]
OK,
